### PR TITLE
Bench 349: Products set to have one category

### PR DESCRIPTION
**Pull request by NogaAD**

### CODE REVIEW

**Review by MichaelOSullivanAnswer — APPROVED** [review]

**Comment by JoeCSykes** [comment]
not sure why but when I run the server and try to hit any endpoints I get the following error
image

File: answerking_app/tests/fixtures/products/basic-1-different-name.json [hunk 1]
```diff
@@ -3,6 +3,6 @@
     "name": "BurgerTwo",
     "description": "desc",
     "price": 1.2,
-    "category": [],
```
Comment (JoeCSykes) [inline answerking_app/tests/fixtures/products/basic-1-different-name.json:6]: These are used by the integration tests and will be replaced so dont' worry about changing these, just leave them as they are and I will change them in my PR.

File: answerking_app/tests/fixtures/products/basic-1.json [hunk 2]
```diff
@@ -3,6 +3,6 @@
     "name": "Burger",
     "description": "desc",
     "price": 1.2,
-    "category": [],
```
Comment (JoeCSykes) [inline answerking_app/tests/fixtures/products/basic-1.json:6]: These are used by the integration tests and will be replaced so dont' worry about changing these, just leave them as they are and I will change them in my PR.

File: answerking_app/tests/test_unit/test_unit_lineitem_product_serializer.py [hunk 3]
```diff
@@ -87,22 +86,6 @@ def test_product_read_only_serializer_price_content(self):
         actual: Decimal = test_serializer_data["price"]
         self.assertEqual(actual, expected)
 
-    def test_product_read_only_serializer_categories_content(self):
```
Comment (JoeCSykes) [inline answerking_app/tests/test_unit/test_unit_lineitem_product_serializer.py:90]: does the product_read_only_serializer not exist any more?

**Review by JoeCSykes — APPROVED** [review]

**Pull request merged**